subsys/bluetooth/mesh/shell.c

```diff
@@ -1705,7 +1705,6 @@ static int cmd_mod_pub(const struct shell *shell, size_t argc, char *argv[])
 	if (argc == 1 || argc == 8) {
 		cid = strtoul(argv[0], NULL, 0);
 		argc--;
-		argv++;
```

Comment:
What's the reasoning behind this change? The original looks correct to me

Reply:
`argv += 3;` so now` argv[0] = pub.addr`, but if we continue increment by one, argv[0] will lost first params of publish set.

Reply:
sorry, maybe i'm wrong.